dashboard/app/assets/javascripts/header.js.erb

```diff
@@ -91,6 +93,26 @@ dashboard.buildHeader = function (stageData, progressData, currentLevelId) {
   $('.header_popup').click(function (e) {
     e.stopPropagation(); // Clicks inside the popup shouldn't close it
   });
+  $('.header_popup_close').click(hideHeaderPopup);
+
+  $(window).resize(function () {
+    if ($('.header_popup').is(':visible')) {
```

Comment:
If you want to check it out, I just submitted an update that fundamentally does this but wraps it up as a debounce function.  I'm not sure what utilities we have access to from this file, or I'd say we should just use lodash's debounce.